### DISCUSSION TITLE
Community meeting on 9th May

### DISCUSSION THREAD

**Discussion by pavithraes** [nebari-dev/nebari#1795] [Community meetings]
We'll have a community meeting in about an hour, please add any discussion topics and updates to the meeting notes shared below. :)

🕒 Meeting time: 2-3 pm GMT

💻 Meeting room: [meet.google.com/rio-ngrj-kyh](https://meet.google.com/rio-ngrj-kyh)

📝 Meeting notes: https://hackmd.io/@nebari/HkH2ypvVn

🗓️ Meeting calendar: [nebari.dev/docs/community#community-meetings](https://www.nebari.dev/docs/community#community-meetings)

**Comment by ghost** [comment]
Hello!
